src/smif/sample_project/config/project.yml

```diff
@@ -1,69 +1,85 @@
 name: Test Project
-scenario_sets:
-- description: Growth in UK population
-  facets:
+scenarios:
   - name: population
-    description: ''
-  name: population
-- description: Rainfall in the UK
-  facets:
-  - name: raininess
-    description: ''
-  name: rainfall
-narrative_sets:
-- description: Describes the evolution of technology
-  name: technology
-interval_definitions:
-- description: ''
-  filename: annual_intervals.csv
-  name: annual
+    description: Growth in UK population
```

Comment:
This looks good, but a few comments:
- Is year (timestep) assumed as a default dim?
- Mohammad uses a different term to "variants" - "replicates". Should we use that? 
- I understand the `provides` heading to contain a Spec? 

Please add a description of this new format to the docs.

Under `variants | data` why is there a list for data?

Reply:
Documentation: TODO..

Timestep is currently assumed to exist in the data (or is picked up if present), but is not included in the spec because we assume that models only request data in the context of a timestep (so they don't expect it as a dimension). It does seem a little tricky - almost an optional dimension. We do see some cases where multi-timestep data is requested for the whole simulation period, e.g. (I think) for energy demand floor area data.

Variants is my best attempt at avoiding `_sets`. A `scenario` here has `variants` exactly as a `scenario_set` had `scenarios` previously. So we can have a population scenario with high/baseline/low variants, or an economic scenario with HHC/LLC/CCH/CCL... variants.  Happy to roll back or change the wording if it's not helpful.

Replicates would be good candidates for a dimension: a river flows scenario could provide 100 replicates of flows at each river gauge in Ml/day. Dims would be `['gauge', 'day', 'replicate']`. Otherwise we'd have a list of variants like `['near-future_0', 'near-future_1', ... 'far-future_0', 'far-future_1' ... ]` each with dims `['gauge', 'day']`.

Each scenario_set (scenario) `provides` a list of outputs, each defined with a `Spec`, exactly.

Each scenario (variant) must then provide data for each of those outputs. The `dict` under `data` must have a key for each variable name, and the value must be the filename where that variable data can be found.